README.md

```diff
@@ -1,6 +1,6 @@
 # Composables specification
 
-## Introduction
+## Compatibility between different implementations
```

Comment:
I don't think this should be at the heading level. Instead, try:

## Introduction

The composables specification exists in order to define a standard format for composable factory functions (called **stamps**). It exists to ensure compatability between different stamp implementations.